ui/plugins/failure_summary/controller.js

```diff
@@ -76,5 +76,51 @@ treeherder.controller('BugsPluginCtrl', [
                 });
             }
         };
+
+        $scope.fileBug = function(event) {
+            var target = event.target;
+            // Click target is sometimes the icon in the button, not the button
+            if(target.tagName.toLowerCase() == "i" && target.className.search("fa-bug") >= 0) {
+                target = target.parentNode;
+            }
+            var summary = target.nextElementSibling.textContent;
+            var allFailures = [];
+            var allFailuresFinder = $(".failure-summary-list .job-tabs-content");
+
+            for( var i=0; i<allFailuresFinder.length; i++) {
+                allFailures.push(allFailuresFinder[i].textContent.trim().split(" | "));
+            }
+
+            var modalInstance = $modal.open({
```

Comment:
I was using $modal because that's what was in use when I first started this patch waaaaay back last year... Easy to switch it to $uibModal, though. Fixed. I don't see that message in the console anymore, unless I'm looking at the wrong time.